volatility3/framework/plugins/windows/modules.py

```diff
@@ -268,26 +308,29 @@ def find_session_layer(
     def list_modules(
         cls,
         context: interfaces.context.ContextInterface,
-        layer_name: str,
-        symbol_table: str,
+        kernel_module_name: str,
     ) -> Iterable[interfaces.objects.ObjectInterface]:
         """Lists all the modules in the primary layer.
 
         Args:
             context: The context to retrieve required elements (layers, symbol tables) from
-            layer_name: The name of the layer on which to operate
-            symbol_table: The name of the table containing the kernel symbols
-
+            kernel_module_name: The name of the module for the kernel
         Returns:
             A list of Modules as retrieved from PsLoadedModuleList
         """
 
-        kvo = context.layers[layer_name].config.get("kernel_virtual_offset", None)
+        kernel = context.modules[kernel_module_name]
+
+        kvo = context.layers[kernel.layer_name].config.get(
```

Comment:
At some point this should become kernel.offset, but I don't think we can guarantee that's the case yet.  One to remember though...